doc/classes/InputEventKey.xml

```diff
@@ -76,6 +77,7 @@
 		</member>
 		<member name="keycode" type="int" setter="set_keycode" getter="get_keycode" enum="Key" default="0">
 			Latin label printed on the key in the current keyboard layout, which corresponds to one of the [enum Key] constants.
+			Key codes are intended for use in standard Latin keyboard shortcuts, e.g., Ctrl+S for Save.
```

Comment:
While we're at it, we _could_ remove "which corresponds to one of the [enum Key] constants" since https://github.com/godotengine/godot/pull/106159 . Unsure if the enum link has a subtle, additional meaning to it, though. It is also parallel with `key_label`'s description, so maybe it's better if it stays.
```suggestion
			Latin label printed on the key in the current keyboard layout, which corresponds to one of the [enum Key] constants. Key codes are meant for shortcuts expressed with a standard Latin keyboard, such as [kbd]Ctrl + S[/kbd] for a "Save" shortcut.
```
Or similar? Not sure about "expressed" in particular.